app/mailers/user_mailer.rb

```diff
@@ -3,6 +3,7 @@
 class UserMailer < ApplicationMailer
   MENTOR_WELCOME_TEMPLATE = "c0dc7dae-76e3-4346-8c2b-0d38aaf94a54"
   ECT_WELCOME_TEMPLATE = "652fea63-1344-4608-a957-6046dc27120b"
+  SIGN_IN_EMAIL_TEMPLATE = "a3219fe5-e320-48ee-a386-7a244785785c"
```

Comment:
What's the string in this relate to? A template you'd expect to see Dear X etc. Is a like a reference to a template in notify?

Reply:
It relates to https://www.notifications.service.gov.uk/services/ea6cc52b-8cba-4568-9293-14d7049bc753/templates/a3219fe5-e320-48ee-a386-7a244785785c